api/CanvasRenderingContext2D.json

```diff
@@ -3272,10 +3328,22 @@
               "version_added": false
             },
             "opera": {
-              "version_added": true
+              "version_added": true,
+              "flags": [
```

Comment:
OK, this flag mirroring is good because it makes it clear it's not supported by default, which the previous data implied.